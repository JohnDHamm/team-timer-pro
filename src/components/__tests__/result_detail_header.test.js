import React from 'react'
import renderer from 'react-test-renderer'
import ResultDetailHeader from '../result_detail_header'

describe('<ResultDetailHeader />', () => {
  const workout = {
    discipline: 'swim',
    description: 'Jun 13 - 4 x 100yd'
  };

  test('it renders correctly', () => {
    const tree = renderer.create(<ResultDetailHeader workout={workout}/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
});
