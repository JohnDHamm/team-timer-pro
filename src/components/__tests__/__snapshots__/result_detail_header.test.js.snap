// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultDetailHeader /> it renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 13.468013468013467,
        "tintColor": "#BADA55",
        "width": 20,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#BADA55",
        "fontFamily": "dosis-medium",
        "fontSize": 20,
        "paddingLeft": 5,
      }
    }
  >
    Jun 13 - 4 x 100yd
  </Text>
</View>
`;
